Gemfile

```diff
@@ -56,3 +58,9 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 group :production do
   gem 'pg', '0.18.4'
 end
+
+# replaces npm uglifier which does not work on heroku
+gem 'uglifier'
```

Comment:
Why not? Is it something like [this](https://github.com/browserify-rails/browserify-rails/issues/137) or something else?